flats/flatsscraper/README.md

```diff
@@ -0,0 +1,55 @@
+# scrappy
```

Comment:
Ten cały flatscrapper niech będzie poprostu dodatkowym spiderem w folderze tricity/tricity/spiders/ .

Nie jest potrzebny dodatkowy folder na to oraz dodatowa struktura scrappiego

Reply:
ok

Reply:
ymm tylko pytanie ....co z plikiem settings - bo tam są podane nazwy do jakich kolekcji maja być zescrapowane mieszaknia i domy ...czy zmienić nazwy settings na osobne jakoś je rozserzyć - bo chyba w jednym nie da sie tego zapisać .... jak sie zdzwonimy wytłumaczę o co chodzi